### PR TITLE
Disable delete button when no checkbox is selected

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts [hunk 1]
```diff
@@ -91,11 +95,57 @@ export class TransactionsInErrorTableDataSource extends TableDataSource<Transact
     });
   }
 
+  public toggleRowSelection(row: Data, event: MatCheckboxChange) {
```
Comment (LucasBrazi06) [inline src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts:98]: Call super class and add just the line to disable the button

File: src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts [hunk 2]
```diff
@@ -91,11 +95,57 @@ export class TransactionsInErrorTableDataSource extends TableDataSource<Transact
...
+      if (event.checked) {
+        row.isSelected = event.checked;
+        this.selectedRows = 1;
+        this.lastSelectedRow = row;
+      }
+    }
+    this.deleteManyAction.disabled = !(this.selectedRows > 0);
+  }
+
+  public selectAllRows() {
```
Comment (LucasBrazi06) [inline src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts:119]: Same as above

File: src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts [hunk 3]
```diff
@@ -91,11 +95,57 @@ export class TransactionsInErrorTableDataSource extends TableDataSource<Transact
...
+    this.data.forEach((row) => {
+      if (row.isSelectable) {
+        row.isSelected = true;
+        this.selectedRows++;
+      }
+    });
+    this.deleteManyAction.disabled = !(this.selectedRows > 0);
+  }
+
+  public clearSelectedRows() {
```
Comment (LucasBrazi06) [inline src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts:131]: Same as above

File: src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts (hunk 1 above)
Comment (mahdibenromdhane) [reply src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts:98]: Done

File: src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts (hunk 2 above)
Comment (mahdibenromdhane) [reply src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts:103]: Done

File: src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts (hunk 3 above)
Comment (mahdibenromdhane) [reply src/app/pages/transactions/in-error/transactions-in-error-table-data-source.ts:108]: Done

**Pull request merged**